### PR TITLE
[Refactor]: Height for log explorer list panel

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
Fix the height issue for logs.

https://github.com/SigNoz/signoz/assets/26198062/30ccedb0-010a-4dbb-8680-d0d4aaa5a70d

### CODE REVIEW

**Review by Vikrant2520 — APPROVED** [review]

**Review by Vikrant2520 — APPROVED** [review]

File: frontend/src/components/Logs/TableView/config.ts [hunk 1]
```diff
@@ -14,12 +14,12 @@ export function getDefaultCellStyle(isDarkMode?: boolean): CSSProperties {
 		lineHeight: '18px',
 		letterSpacing: '-0.07px',
 		marginBottom: '0px',
+		minWidth: '10rem',
```
Comment (YounixM) [inline frontend/src/components/Logs/TableView/config.ts:17]: ??

**Comment by YounixM** [comment]
@Rajat-Dabade: Please add an RCA around. It will help understanding the internal functioning and requirement for change.

File: frontend/src/components/Logs/TableView/config.ts (hunk 1 above)
Comment (Rajat-Dabade) [reply frontend/src/components/Logs/TableView/config.ts:17]: This is not super important, but it looks cleaner with minWidth and also we have a horizontal scroll, so there will be no problem. 

With MinWidth: 

https://github.com/SigNoz/signoz/assets/26198062/e8b1715c-df65-4de0-a0fb-4d03b9b23b99

Without minWidth:

https://github.com/SigNoz/signoz/assets/26198062/a11b8376-0edc-49f1-8430-a7aeecac779f

**Comment by Rajat-Dabade** [comment]
Root Cause Analysis (RCA):

Issue Description:
The last `<div>` element within a flex container does not take up the full remaining height, despite setting `display: flex;` and `flex-direction: column;` on the container.

As you can see in the image the last `div` is not taking the full height available. 

Screenshot 2024-03-11 at 3 03 43 PM




Root Cause:

The reason for the last `<div>` not taking the full remaining height within the flex container is due to the default behaviour of flex items in a flex container. By default, flex items do not automatically stretch to fill the remaining space along the cross-axis (in this case, the vertical axis since the flex-direction is set to column).

Analysis:
Flexbox Behavior: In a flex container with `flex-direction: column;` , the main axis runs vertically from top to bottom, while the cross-axis runs horizontally. By default, flex items only occupy the height required by their content along the cross-axis.

Lack of Flex Grow Property: Without explicitly specifying the `flex: 1` property on the last `<div>` element, it does not have instructions to expand and fill the remaining space within the flex container along the cross-axis.

Solution:

To address the issue and ensure that the last `<div>` takes up the full remaining height within the flex container, the `flex: 1` property needs to be applied to the last `<div>` element. Setting `flex: 1;` set `flex-grow: 1`  on the last <div> instructs it to expand and fill the remaining space along the cross-axis, thereby resolving the issue.

**Review by YounixM — APPROVED** [review]

**Pull request merged**